### ISSUE TITLE
Better error message support for `yo-code`

### ISSUE THREAD

**Issue by parthsarthiprasad** [microsoft/vscode-generator-code#198]
I've been having this issue with generator where It throws off an error code without any information about the error message. Rather than just `Error code`
 If there can be development in field of error messages being hit either due to `wrongurl` or `wrongfilepath` which can be thrown will be helpful for the use case?

#snippet of the issue
```
     _-----_     ╭──────────────────────────╮
    |       |    │   Welcome to the Visual  │
    |--(o)--|    │   Studio Code Extension  │
   `---------´   │        generator!        │
    ( _´U`_ )    ╰──────────────────────────╯
    /___A___\   /
     |  ~  |     
   __'.___.'__   
 ´   `  |° ´ Y ` 

? What type of extension do you want to create? New Language Support
Enter the URL (http, https) or the file path of the tmLanguage grammar or press ENTER to start with a new grammar.
? URL or file to import, or none for new: https://github.com/parthsarthiprasad/chapel-tmbundle/blob/tmbundlequotes/Syntaxes/Chapel.tmLanguage
Error code 

undefined
```
or using rawgithubusercontent

```
  _-----_     ╭──────────────────────────╮
    |       |    │   Welcome to the Visual  │
    |--(o)--|    │   Studio Code Extension  │
   `---------´   │        generator!        │
    ( _´U`_ )    ╰──────────────────────────╯
    /___A___\   /
     |  ~  |     
   __'.___.'__   
 ´   `  |° ´ Y ` 

? What type of extension do you want to create? New Language Support
Enter the URL (http, https) or the file path of the tmLanguage grammar or press ENTER to start with a new grammar.
? URL or file to import, or none for new: https://raw.githubusercontent.com/parthsarthiprasad/chapel-tmbundle/tmbundlequotes/Syntaxes/Chapel.tmLanguage
Error code 

undefined
```

[https://raw.githubusercontent.com/parthsarthiprasad/chapel-tmbundle/tmbundlequotes/Syntaxes/Chapel.tmLanguage](url)

**Comment by parthsarthiprasad** [comment]
Do let me know about the possibility and also I'll be happy to contribute for the same if possible(If there are good first issues, I'd also like to try them)

**Comment by aeschli** [comment]
fixed by 4e72a487f78dcae20fc100090cbed3f3cd9b8357
